### PR TITLE
memsql: fix memsql active state check.

### PR DESCRIPTION
**Pull request by sashas-anyvision**
Currently we checked memsql active state by running show dbs command.
However we got a failure:
'Partition tracks_db:0 has no master instance. This is likely because the node or nodes that hold a copy of the partition are down. Check for offline leaf nodes by running SHOW LEAVES and bring them back online to restore access to the partition')

This is probably happening due to the incorrect check of "memsql" up.
Changed this to check the leaves state

### CODE REVIEW

File: automation/devops_automation_infra/plugins/memsql.py [hunk 1]
```diff
@@ -173,7 +174,8 @@ def fetch_count(self, query):
 
     def ping(self):
         try:
-            return self.fetch_all("show databases")
+            nodes_status = json.loads(self._host.Docker.run_cmd_in_service('memsql', 'gosu memsql memsql-admin list-nodes --json'))
+            return all([node['processState'] == 'Running' for node in  nodes_status['nodes']])
```
Comment (LazyDBA247-Anyvision) [inline automation/devops_automation_infra/plugins/memsql.py:178]: great! in bash I do the same but uglier :)

**Review by LazyDBA247-Anyvision — APPROVED** [review]

File: automation/devops_automation_infra/plugins/memsql.py (hunk 1 above)
Comment (LazyDBA247-Anyvision) [reply automation/devops_automation_infra/plugins/memsql.py:178]: processState = Running 
Connectable = True
RecoveryState = Online

these are the correct states

Comment (sashas-anyvision) [reply automation/devops_automation_infra/plugins/memsql.py]: should i check all of those states ?

Comment (sashas-anyvision) [reply automation/devops_automation_infra/plugins/memsql.py]: ok i added the checks for all 3

**Comment by sashas-anyvision** [comment]
pr merge is sucessfulll .. only ws scan failed ..

**Pull request merged**